<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>STEMe - Khoa học và giáo dục</string>
  </property>
  <property name="windowIcon">
   <iconset resource="group.qrc">
    <normaloff>:/icon/group/steme_icon.jpg</normaloff>:/icon/group/steme_icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget_online">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>5</y>
      <width>1071</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Type">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>88</y>
      <width>251</width>
      <height>713</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 255, 255);
background-color: rgb(245, 245,245);
background-color: rgb(250, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Group">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="movement">
     <enum>QListView::Static</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::IconMode</enum>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <property name="uniformItemSizes">
     <bool>true</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_groupSoftware">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#toolButton_groupSoftware{
color: rgb(2, 87, 122);
background-color: rgb(240, 240, 240);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 15px;
}
QToolButton#toolButton_groupSoftware:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Phần mềm</string>
    </property>
    <property name="icon">
     <iconset resource="group.qrc">
      <normaloff>:/icon/group/iconSoftwareGroup_off.jpg</normaloff>
      <activeoff>:/icon/group/iconSoftwareGroup_on.jpg</activeoff>:/icon/group/iconSoftwareGroup_off.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_groupSensor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#toolButton_groupSensor{
color: rgb(2, 87, 122);
background-color: rgb(240, 240, 240);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 15px;
}
QToolButton#toolButton_groupSensor:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Cảm biến</string>
    </property>
    <property name="icon">
     <iconset resource="group.qrc">
      <normaloff>:/icon/group/iconSensorGroup_off.jpg</normaloff>
      <activeoff>:/icon/group/iconSensorGroup_on.jpg</activeoff>:/icon/group/iconSensorGroup_off.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_groupSystem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#toolButton_groupSystem{
color: rgb(3, 87, 123);
background-color: rgb(240, 240, 240);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 15px;
}
QToolButton#toolButton_groupSystem:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Mô hình thí nghiệm</string>
    </property>
    <property name="icon">
     <iconset resource="group.qrc">
      <normaloff>:/icon/group/iconSystemGroup_off.jpg</normaloff>
      <activeoff>:/icon/group/iconSystemGroup_on.jpg</activeoff>:/icon/group/iconSystemGroup_off.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_groupModule">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#toolButton_groupModule{
color: rgb(2, 87, 122);
background-color: rgb(240, 240, 240);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 15px;
}
QToolButton#toolButton_groupModule:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Mô đun thực hành</string>
    </property>
    <property name="icon">
     <iconset resource="group.qrc">
      <normaloff>:/icon/group/iconModuleGroup_off.jpg</normaloff>
      <activeoff>:/icon/group/iconModuleGroup_on.jpg</activeoff>:/icon/group/iconModuleGroup_off.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TypeList">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>251</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_TypeList{
color: rgb(255, 255, 255);
background-color: rgb(25, 149, 173);
border: none;
border-width: 0px;
}
QPushButton#pushButton_TypeList:hover{
	background-color: rgb(31, 188, 216);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Hiển thị mã thiết bị</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>101</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>3</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_Display">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>901</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_groupSearch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#toolButton_groupSearch{
color: rgb(2, 87, 122);
background-color: rgb(240, 240, 240);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 15px;
}
QToolButton#toolButton_groupSearch:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Tìm thiết bị</string>
    </property>
    <property name="icon">
     <iconset resource="group.qrc">
      <normaloff>:/icon/group/iconSearchGroup_off.jpg</normaloff>
      <activeoff>:/icon/group/iconSearchGroup_on.jpg</activeoff>:/icon/group/iconSearchGroup_off.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_groupConnection">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#toolButton_groupConnection{
color: rgb(2, 87, 122);
background-color: rgb(240, 30 ,40);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 20px;
}
QToolButton#toolButton_groupConnection:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Kết nối</string>
    </property>
    <property name="icon">
     <iconset resource="group.qrc">
      <normaloff>:/icon/group/iconSoftwareConnect_off.jpg</normaloff>
      <activeoff>:/icon/group/iconSoftwareConnect_Active.jpg</activeoff>:/icon/group/iconSoftwareConnect_off.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>35</height>
     </size>
    </property>
    <property name="popupMode">
     <enum>QToolButton::DelayedPopup</enum>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_login">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#toolButton_login{
color: rgb(2, 87, 122);
background-color: rgb(240, 240, 240);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 15px;
}
QToolButton#toolButton_login:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Đăng nhập</string>
    </property>
    <property name="icon">
     <iconset resource="group.qrc">
      <normaloff>:/icon/group/iconLogin_off.jpg</normaloff>
      <activeoff>:/icon/group/iconLogin_on.jpg</activeoff>:/icon/group/iconLogin_off.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_clearDevice">
    <property name="geometry">
     <rect>
      <x>1195</x>
      <y>5</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#toolButton_clearDevice{
color: rgb(2, 87, 122);
background-color: rgb(240, 240, 240);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 15px;
}
QToolButton#toolButton_clearDevice:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Xóa thiết bị</string>
    </property>
    <property name="icon">
     <iconset resource="group.qrc">
      <normaloff>:/icon/group/iconClear_Off.jpg</normaloff>
      <activeoff>:/icon/group/iconClear_On.jpg</activeoff>:/icon/group/iconClear_Off.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>101</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>toolButton_groupConnection</tabstop>
  <tabstop>listWidget_Type</tabstop>
  <tabstop>listWidget_Group</tabstop>
  <tabstop>toolButton_groupSearch</tabstop>
  <tabstop>toolButton_groupSoftware</tabstop>
  <tabstop>toolButton_groupSensor</tabstop>
  <tabstop>toolButton_groupModule</tabstop>
  <tabstop>toolButton_groupSystem</tabstop>
  <tabstop>toolButton_login</tabstop>
  <tabstop>listWidget_online</tabstop>
  <tabstop>pushButton_TypeList</tabstop>
  <tabstop>toolButton_clearDevice</tabstop>
 </tabstops>
 <resources>
  <include location="group.qrc"/>
 </resources>
 <connections/>
</ui>
